saleor/graphql/order/utils.py

```diff
@@ -118,8 +124,12 @@ def validate_product_is_published(order):
 def validate_product_is_published_in_channel(variants, channel):
     if not channel:
         raise ValidationError(
-            "Can't add product variant for draft order without channel",
-            code=OrderErrorCode.REQUIRED,
+            {
+                "channel": ValidationError(
+                    "Can't add product variant for draft order without channel",
+                    code=OrderErrorCode.REQUIRED,
```

Comment:
What is an expected way for `code`? In some places you're using `ErrorClass.CODE.value` and in other `ErrorClass.CODE`
https://github.com/mirumee/saleor/pull/7158/files#diff-c70a2160c49088fa171fda2b4904744130f9cfa1476711c26813f0cf744f20feR173

Would be good to at least unify the way for single PR